main.js

```diff
@@ -116,6 +124,16 @@ projects.map((project) => {
   professionalArtDiv.appendChild(artContainerDiv);
 });
 
+// populating form value from localStorage
+
+const localStorageData = JSON.parse(localStorage.getItem("formData"));
```

Comment:
- [ ] When the user loads the page, the code checks if there are any saved data in the local storage, and if so the input fields are pre-filled with the data.